components/studentActivity.js

```diff
@@ -0,0 +1,32 @@
+export default function getStudentActivity(props) {
+  const thresholdTime = 604800000; // time of one week in milliseconds
+  let today = Math.floor(new Date().getTime());
+  let recentCompletionCount = 0;
+  let mostRecentCompletionTime = 0;
+
+  for (let i = 0; i < props.recentCompletions.length; i++) {
+    let period = today - props.recentCompletions[i];
+    if (period < thresholdTime) {
+      recentCompletionCount++;
+    }
+    if (mostRecentCompletionTime < props.recentCompletions[i]) {
+      mostRecentCompletionTime = props.recentCompletions[i];
+    }
+  }
+  var mostRecentDate = new Date(mostRecentCompletionTime);
+  let mostRecentDateText =
+    'Last completion time: ' + mostRecentDate.toLocaleString();
+
+  return (
+    <div
+      class={`... ${
```

Comment:
```suggestion
      className={`${
```

The three dots that were in my example were meant to represent any other CSS that you wanted to write for the component, Tailwind will automatically purge this when compiled but we should take it out ourselves instead. NextJS also uses the `className` convention instead of `class`.

Reply:
Thank you for pointing this out! I've removed the '...'